test/babel/fixtures/issue-427/.babelrc

```diff
@@ -0,0 +1,3 @@
+{
```

Comment:
can you call this directory something like `arrow-function-in-constructor` rather than `issue-427`? Also, we could probably use a few more test fixtures, see [what we have for class properties](https://github.com/gaearon/react-hot-loader/tree/next/test/babel/fixtures/class-properties).

I'd also like at least one `AppContainer` test, since those are more integration-level tests that verify that RHL works, not just the Babel plugin. Here's a [reference example](https://github.com/gaearon/react-hot-loader/blob/next/test/AppContainer/AppContainer.dev.js#L244)